packages/infrastructure/kube_airbyte/vars.tf

```diff
@@ -0,0 +1,408 @@
+    condition     = var.pgbouncer_minimum_cpu_millicores >= 10
+    error_message = "Must provide at least 10m of CPU"
+  }
+}
+
+variable "pgbouncer_maximum_cpu_millicores" {
+  description = "The maximum amount of cpu to allocate to the pgbouncer pods (in millicores)"
+  type        = number
+  default     = 10000
+}
+
+variable "db_backup_directory" {
+  description = "Directory to store database backups (if enabled)"
+  type        = string
+  default     = "initial"
+}
+
+variable "db_recovery_mode_enabled" {
+  description = "Whether to enable recovery mode for the database"
+  type        = bool
+  default     = false
+}
+
+variable "db_recovery_directory" {
+  description = "Directory to restore database from (if recovery mode enabled)"
+  type        = string
+  default     = null
+}
+
+variable "db_recovery_target_time" {
+  description = "Target recovery time for the database (if recovery mode enabled)"
+  type        = string
+  default     = null
+}
+
+# Component replicas
+variable "worker_replicas" {
+  description = "Number of worker replicas"
+  type        = number
+  default     = 1
+  validation {
+    condition     = var.worker_replicas >= 1
+    error_message = "The worker_replicas value must be at least 1."
+  }
+}
+
+# Scheduler and node configuration
+variable "panfactum_scheduler_enabled" {
+  description = "Whether to enable the Panfactum scheduler"
+  type        = bool
+  default     = true
+}
+
+variable "pull_through_cache_enabled" {
+  description = "Whether to enable pull-through cache for container images"
+  type        = bool
+  default     = true
+}
+
+variable "arm_nodes_enabled" {
+  description = "Whether to allow scheduling on arm nodes"
+  type        = bool
+  default     = true
+}
+
+variable "spot_nodes_enabled" {
+  description = "Whether to allow scheduling on spot nodes"
+  type        = bool
+  default     = true
+}
+
+variable "burstable_nodes_enabled" {
+  description = "Whether to allow scheduling on burstable nodes"
+  type        = bool
+  default     = true
+}
+
+variable "controller_nodes_enabled" {
+  description = "Whether to allow scheduling on controller nodes"
+  type        = bool
+  default     = false
+}
+
+variable "aws_iam_ip_allow_list" {
+  description = "List of IPs to allow for AWS IAM access"
+  type        = list(string)
+  default     = []
+}
+
+variable "jobs_min_memory_mb" {
+  description = "Memory request for jobs containers"
+  type        = number
+  default     = 1024
+}
+
+variable "jobs_cpu_min_millicores" {
+  description = "The minimum amount of cpu millicores for jobs containers"
+  type        = number
+  default     = 100
+}
+
+variable "jobs_env_env" {
+  description = "Additional environment variables for Airbyte jobs configuration (e.g. SYNC_JOB_MAX_ATTEMPTS, JOB_MAIN_CONTAINER_MEMORY_LIMIT, etc.) https://docs.airbyte.com/operator-guides/configuring-airbyte#jobs"
+  type        = map(string)
+  default     = {}
+}
+
+variable "webapp_min_memory_mb" {
```

Comment:
In some variables, you use `min` / `max`. In others, you use `minimum` / `maximum`. Choose one convention.